app/services/credit_card_service.rb

```diff
@@ -0,0 +1,45 @@
+require 'stripe'
+
+class CreditCardService
+
+	def self.create_stripe_customer(current_user,params)
+		Stripe::Customer.create(
+			email: current_user.email,
+			source: get_token(params),
+			description: [current_user.first_name, current_user.last_name].join('')
```

Comment:
this will join the first and last name together without a space in between the names, is that intended behavior?

Reply:
Nope, corrected!